### PR TITLE
Jsonlh engine api split

### PR DESCRIPTION
**Pull request by chillaq**
# Ruby SDK

### CODE REVIEW

File: lib/splitclient-rb/engine/api/json_localhost/splits.rb [hunk 1]
```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+module SplitIoClient
+  module Api
+    # Retrieves split definitions from the Split Backend
```
Comment (sanzmauro) [inline lib/splitclient-rb/engine/api/json_localhost/splits.rb:5]: 🤔 🤔

File: lib/splitclient-rb/engine/api/json_localhost/splits.rb [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+          parsed = JSON.parse(File.read(@split_file), symbolize_names: true)
+          santitized = sanitize_feature_flag(parsed)
+          return santitized
+        rescue StandardError => e
+          @config.logger.error("Exception caught: #{e.message}")
+          raise "Error parsing splits file \'#{@split_file}\', Make sure it's readable."
+        end
+      end
+
+      def sanitize_feature_flag(parsed)
```
Comment (sanzmauro) [inline lib/splitclient-rb/engine/api/json_localhost/splits.rb:58]: maybe you can create a `splits_sanitizer.rb` and move all this methods

File: lib/splitclient-rb/engine/api/json_localhost/splits.rb (hunk 1 above)
Comment (chillaq) [reply lib/splitclient-rb/engine/api/json_localhost/splits.rb:5]: This class is used in the engine in the same way the Api classes used (for http and redis synching), that is why I added it here.

File: lib/splitclient-rb/engine/api/json_localhost/splits.rb (hunk 2 above)
Comment (chillaq) [reply lib/splitclient-rb/engine/api/json_localhost/splits.rb:58]: Sure, will do.

**Review by sanzmauro — APPROVED** [review]

**Pull request merged**